azure-mgmt-batchai/azure/mgmt/batchai/operations/clusters_operations.py

```diff
@@ -133,7 +137,7 @@ def get_long_running_output(response):
             get_long_running_status, long_running_operation_timeout)
 
     def update(
-            self, resource_group_name, cluster_name, parameters, custom_headers=None, raw=False, **operation_config):
+            self, resource_group_name, cluster_name, tags=None, scale_settings=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
@AlexanderYukhanov This is because you didn't use `payload-flattening-threshold=2`. Please update your tests if necessary.

Reply:
Thank you, will do